town.logic

```diff
@@ -1,2 +1,46 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node FigurineHouseDoorEast:
+      door FigurineHouseEast
+    node Main:
+      chest CafeLady
+      door Bakery
+      door East
+      door FigurineHouseWest
+      door HearthLedge
+      door HearthNorth
+      door HearthSOuth
+      door Julietta
+      door MayorEast
+      door MayorWest
+      door North
+      door Rem
+      door SchoolEast
+      door SchoolWest
+      door Simon
+      door South
+      door Stockwell
+      door Swiftblade
+      door Well
+      door West
+      entrance Crest
+      entrance TownDigging
+    Main <-> FigurineHouseDoorEast: item CarlovMedal
+    node MainBellHP:
+      chest MainBellHP
+    Main -> MainBellHP: item Cape 
+    node MainMinish
+    Main <-> MainMinish: item CaneOfPacci
+    node NorthEastMinish:
+      door SchoolEntranceMinishEast
+      door SchoolGarden
+    node SouthWest:
+      door Carlov
+      door Carpenters
+      door DrLeft
+    Main <-> SouthWest: (item Flippers | item Cape)
```

Comment:
This needs a case for the dog to move out of the way - a flag that can then also be said as minish in this area.

Reply:
So something like flag MinishInThisArea?

Reply:
I'd name it something like `flag DogMoves` but yeah.

Reply:
Alright!